CMakeLists.txt

```diff
@@ -541,10 +558,16 @@ enable_language(C)
 #define FORTRAN FLAGS
 set(CMAKE_F_FLAGS -std=c99 CACHE STRING "Fortran Flags")
 
+enable_language(Fortran)
+if(QUDA_MPI OR QUDA_QMP)
+  set(CMAKE_Fortran_COMPILER mpifort)
```

Comment:
again don't set compilers here

Reply:
The problem I was having here (related to the above comment too) was that when I performed a QMP or MPI build, gfortran was selected as the fortran compiler. Changing this forced me to delete the cache and start over:

```
You have changed variables that require your cache to be deleted.
 Configure will be re-run and you may have to reset some variables.
 The following variables have changed:
 CMAKE_Fortran_COMPILER=mpifort
```

Is there any way one can make CMake recognize a multi GPU build and select a default MPI fortran compiler?